front-end/src/Views/TermsOfServicesPage.js

```diff
@@ -0,0 +1,208 @@
+import React, { Component } from "react";
+import "../App.css"
+import axios from 'axios';
+import { styled } from '@material-ui/core';
+import AlertTypes from '../Common/AlertTypes.json';
+import CustomSnackbar from '../Components/CustomSnackbar/CustomSnackbar';
+import PrimarySearchAppBar from '../Components/TopBar/PrimarySearchAppBar';
+import config from "../config";
+import Box from '@material-ui/core/Box';
+import { theme } from "../Common/ColorTheme";
+import Typography from '@material-ui/core/Typography';
+
+
+const Messages = {
+    emptyFieldError: "Please Fill All Areas!",
+    registerSuccess: "Registration Successful! You'll be redirected to login.",
+    somethingWrong: "Something Went Wrong!",
+    emptyNameError: "Please enter your name and surname."
+
+}
+const Container = styled(Box)({
+    // background: theme.palette.secondary.light,
+    background: "white",
+    border: 0,
+    borderRadius: 3,
+    height: "100vh",
+    width: "100%",
+    margin: "auto",
+    '& .MuiTextField-root': {
+        margin: "10px",
+        width: "30%",
+        minWidth: "250px"
+    }
+});
+
+export default class RegistrationPage extends Component {
+
+    constructor(props) {
+        super(props);
+        this.SnackbarRef = React.createRef();
+        this.state = {
```

Comment:
Please delete unused state variables such as firstName middleName etc. Since this page only shows terms of service we don't need much of these.